Firmware/tmc2130.h

```diff
@@ -1,12 +1,14 @@
 #ifndef TMC2130_H
 #define TMC2130_H
 
+#include <stdint.h>
 
 //mode
 extern uint8_t tmc2130_mode;
+// Beware - the 4th index (counted from zero) is abused for the E-motor cool mode only used on the farm
 //holding and running currents
-extern uint8_t tmc2130_current_h[4];
-extern uint8_t tmc2130_current_r[4];
+extern uint8_t tmc2130_current_h[5];
+extern uint8_t tmc2130_current_r[5];
```

Comment:
This is not workable solution. If done this way, you would need to add condition to use index 3 or 4 on every place where this value is used, not only in tmc2130_init(). There are about 16 places in code where this value is used.

More workable solution would be to use approach already used in farm_mode (preferably use named constant instead of plain "36"):
```
		//increased extruder current (PFW363)
		tmc2130_current_h[E_AXIS] = 36;
		tmc2130_current_r[E_AXIS] = 36;
```